tests/system/source/main.d

```diff
@@ -137,8 +137,9 @@ private void assertBlockHeight (const string[] addresses, ulong height)
             }
         }
         while (getHeight < height && times++ < 50); // Retry if we're too early
-        assert(getHeight == height,
-            format!"%s Client #%s still has block height %s not %s"
+        if (idx > 0)
```

Comment:
This seems like a workaround. But it's not documented.

So the FullNode for some reason never gets to height 1+?

Reply:
Forgot to revert this back. Will do.

Reply:
Updated